src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb

```diff
@@ -2309,6 +2309,52 @@ End Class"
             locals.Free()
         End Sub
 
+        <WorkItem(1115044, "DevDiv")>
+        <Fact>
+        Public Sub CaseSensitivity()
```

Comment:
I certainly didn't add that test.  If I can't find coverage in an existing test, then I will.